doc/datastorage_setup

```diff
@@ -0,0 +1,36 @@
+Installing ZeroMQ:
+
+# run in sudo
+# Before installing, make sure you have installed all the needed packages
+1.sudo apt-get install libtool pkg-config build-essential autoconf automake
+
+2.sudo apt-get install libzmq3-dev
+
+# Install libsodium
+1. Download the tar from https://libsodium.gitbook.io/doc/installation
+2. tar -xvf tar -xvf libsodium_1.0.18.orig.tar.gz 
+3. cd libsodium-1.0.18
+4. ./autogen.sh
+5. ./configure
+6. make && make check
+7. sudo make install
+8. sudo ldconfig
+
+# Install zeromq
+1. Download the tar from https://github.com/zeromq/libzmq/tags/v4.2.2
+2. tar xvf libzmq-4.2.2.tar.gz
+3. cd libzmq-4.2.2/
+4. ./autogen.sh
+5. sudo apt-get update
+6. sudo apt-get install -y libtool pkg-config build-essential autoconf automake uuid-dev
+7. ./configure
+8. sudo make install
+9. sudo ldconfig
+# Check installed
```

Comment:
Better to put an blank line between Line 28 and 29. What do you think?

Reply:
Done